src/components/Tool/FixPhoneNumbers/ContactPhoneNumbers.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import { mdiDelete, mdiLock } from '@mdi/js';
+import { Icon } from '@mdi/react';
+import StarIcon from '@mui/icons-material/Star';
+import StarOutlineIcon from '@mui/icons-material/StarOutline';
+import {
+  Box,
+  FormControl,
+  FormHelperText,
+  Grid,
+  Hidden,
+  TextField,
+  Theme,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import clsx from 'clsx';
+import { Field } from 'formik';
+import { DateTime } from 'luxon';
+import { useTranslation } from 'react-i18next';
+import { makeStyles } from 'tss-react/mui';
+import { useLocale } from 'src/hooks/useLocale';
+import { dateFormatShort } from 'src/lib/intlFormat';
+import { isEditableSource, sourceToStr } from 'src/utils/sourceHelper';
+import theme from '../../../theme';
+import { PhoneNumber } from './Contact';
+import { PersonInvalidNumberFragment } from './GetInvalidPhoneNumbers.generated';
+
+const useStyles = makeStyles()((theme: Theme) => ({
```

Comment:
`theme ` is already defined.
```suggestion
const useStyles = makeStyles()(() => ({
```